<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tower_one_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Move top  block to</string>
    </property>
   </widget>
   <widget class="QLabel" name="tower_two_label">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>550</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Move top block to</string>
    </property>
   </widget>
   <widget class="QLabel" name="tower_three_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Move top block to</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="block_number_spin_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="block_number_info_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set the number of blocks:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_push_button">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_push_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_to_A_push_button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>580</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tower 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_to_C_push_button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>615</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tower 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_to_A_push_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>580</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tower 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_to_B_push_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>615</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tower 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_to_B_push_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>580</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tower 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_to_C_push_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>615</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tower 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="timer_lcd_number">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit_push_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
